### PR TITLE
[CHORE] Serializer spec

### PR DESCRIPTION
**Pull request by tochman**
Not documented in PT

* adds an alternative way to check for JSON returned by serializer using serializer specs for a collection
* adds Faker to generate unique emails in User factory

### CODE REVIEW

File: spec/serializers/requests/index_serializer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+RSpec.describe Request::IndexSerializer, type: :serializer do
+  let!(:requests) { 5.times { create(:request) } }
+  let(:serialization) do
+    ActiveModelSerializers::SerializableResource.new(
```
Comment (kermit-klein) [inline spec/serializers/requests/index_serializer_spec.rb:7]: this syntax is not familiar for me, could you explain what it does?

Comment (tochman) [reply spec/serializers/requests/index_serializer_spec.rb:7]: It's basically a version of `each_serializer` that you use when sending off your collection as soon in a controller action.

File: spec/factories/users.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :user do
-    email { "MyString@mail.com" }
+    email { Faker::Internet.email }
```
Comment (ViaMarcus) [inline spec/factories/users.rb:5]: This is very useful, forced to do some workarounds due to requests creating users before

**Review by ViaMarcus — APPROVED** [review]
I like the compact key checking syntax

**Review by erikbjoern — APPROVED** [review]

File: spec/factories/users.rb (hunk 2 above)
Comment (tochman) [reply spec/factories/users.rb:5]: Yes, read more on Faker at: https://github.com/faker-ruby/faker

**Review by kermit-klein — APPROVED** [review]

**Review by PaulineBA — APPROVED** [review]

**Pull request merged**